### ISSUE TITLE
Extension setting "perl.perlInc" not appending paths to @INC

### ISSUE THREAD

**Issue by DerekBoucher** [richterger/Perl-LanguageServer#69]
I'm currently attempting to append paths to `@INC` via the `perl.perlInc` setting as so:
`    "perl.perlInc": [
        "/home/derek.boucher/perl5/lib/perl5/x86_64-linux-gnu-thread-multi"
    ],`

However the extension crashes upon initialisation due to missing libraries:

```
`BEGIN failed--compilation aborted.
[Info  - 12:51:50 p.m.] Connection to server got closed. Server will restart.
Can't locate Class/MOP/Method/Meta.pm in @INC (you may need to install the Class::MOP::Method::Meta module) (@INC contains: /etc/perl /usr/local/lib/x86_64-linux-gnu/perl/5.26.1 /usr/local/share/perl/5.26.1 /usr/lib/x86_64-linux-gnu/perl5/5.26 /usr/share/perl5 /usr/lib/x86_64-linux-gnu/perl/5.26 /usr/share/perl/5.26 /usr/local/lib/site_perl /usr/lib/x86_64-linux-gnu/perl-base) at /usr/share/perl/5.26/Class/MOP/Mixin/HasMethods.pm line 7.
BEGIN failed--compilation aborted at /usr/share/perl/5.26/Class/MOP/Mixin/HasMethods.pm line 7.
Compilation failed in require at /usr/share/perl/5.26/Class/MOP.pm line 16.
BEGIN failed--compilation aborted at /usr/share/perl/5.26/Class/MOP.pm line 16.
Compilation failed in require at /usr/share/perl/5.26/Moose/Exporter.pm line 8.
BEGIN failed--compilation aborted at /usr/share/perl/5.26/Moose/Exporter.pm line 8.
Compilation failed in require at /usr/share/perl/5.26/Moose.pm line 15.
BEGIN failed--compilation aborted at /usr/share/perl/5.26/Moose.pm line 15.
Compilation failed in require at /usr/share/perl/5.26/Perl/LanguageServer.pm line 6.
BEGIN failed--compilation aborted at /usr/share/perl/5.26/Perl/LanguageServer.pm line 6.
Compilation failed in require.`
```

I have Perl::LanguageServer and all of its dependencies installed in `"/home/derek.boucher/perl5/lib/perl5/x86_64-linux-gnu-thread-multi"`

Any clues as to why the extension refuses to append my custom path to @INC?

**Comment by benpm** [comment]
I'm having a similar issue on MacOS Catalina 10.15.7. The extension cannot find the language server module even though I've appended the correct lib directory to the `perl.perlInc` list.
```
$ ls /Users/bmastripolito/local/lib/perl5/Perl
LanguageServer    LanguageServer.pm
```
in settings.json:
```
"perl.perlInc": [
    "/Users/bmastripolito/local/lib/perl5/"
]
```
Perl language server output:
```
Can't locate Perl/LanguageServer.pm in @INC (you may need to install the Perl::LanguageServer module) (@INC contains: /usr/local/Cellar/perl/5.32.0/lib/perl5/site_perl/5.32.0/darwin-thread-multi-2level /usr/local/Cellar/perl/5.32.0/lib/perl5/site_perl/5.32.0 /usr/local/Cellar/perl/5.32.0/lib/perl5/5.32.0/darwin-thread-multi-2level /usr/local/Cellar/perl/5.32.0/lib/perl5/5.32.0 /usr/local/lib/perl5/site_perl/5.32.0/darwin-thread-multi-2level /usr/local/lib/perl5/site_perl/5.32.0).
BEGIN failed--compilation aborted.
```

**Comment by richterger** [comment]
Fixed in b463dc87b6de7b248f96dd2ed29ecb5af3106ff9
